### PR TITLE
Resprites, renames, and redescribes aloe and comfrey poultices. Also adds sounds to the aloe poultice.

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Renames "Aloe Vera leaf" to "Aloe Vera poultice"
Renames "Comfrey Leaf" to "Comfrey poultice"

The descriptions of both items has been changed to be more informative.

Instead of reusing the sprite of the leaf dropped from their respective plants, they now use dedicated sprites to clearly signal that they are medical items (special thanks goes to @McRamon for making the sprites, they came out way nicer than the ones I cooked up).

(NPF) all medical item icons found in **items.dmi** have been relocated to the newly created **medical.dmi**. Stuff pointing to the old icons have had their pointers adjusted accordingly.

Alloe poultices now make sounds like the comfrey poultices do.

## Why It's Good For The Game
New sprites and descriptions let players know these items are for healing instead of just being random leaves.

Sound is soul.

## Images of changes
Poultice preview

## Testing
Appeared.
Saw that the poultices have new names and descriptions and sprites.
Smeared an aloe poultice on a skrell and then threw a poultice at them to ensure the sound was made.

## Changelog
:cl:
tweak: Comfrey and Aloe poultices have new names and descriptions and sprites.
soundadd: Newly renamed Aloe Vera poultices now make sounds when thrown or used.
/:cl:

### CODE REVIEW

**Comment by McRamon** [comment]
Sprites are a little questionable

File: code/game/objects/items/stacks/medical_packs.dm [hunk 1]
```diff
@@ -268,12 +268,11 @@
...
-	name = "\improper Comfrey leaf"
-	singular_name = "Comfrey leaf"
-	desc = "A soft leaf that is rubbed on bruises."
-	icon = 'icons/obj/hydroponics/harvest.dmi'
-	icon_state = "tea_aspera_leaves"
-	color = "#378C61"
+	name = "\improper Comfrey poultice"
+	singular_name = "Comfrey poultice"
+	desc = "A medical poultice for treating brute injuries, made from crushed comfrey leaves. The effectiveness of the poultice depends on the potency of the comfrey it was made from."
+	icon = 'icons/obj/items.dmi'
```
Comment (Henri215) [inline code/game/objects/items/stacks/medical_packs.dm:274]: 1. Find a place that would fit it, not items.dmi (probably keep it on harvest.dmi??).

2. The old sprites are still being used by some other item?

**Review by DGamerL — COMMENTED** [review]
I agree with Henri, please move the sprites to a different file, harvest.dmi still seems appropriate

I also agree with McRamon about the sprite quality, maybe you could ask him in discord for some tips?

File: code/game/objects/items/stacks/medical_packs.dm (hunk 1 above)
Comment (Fordoxia) [reply code/game/objects/items/stacks/medical_packs.dm:274]: 1. Made medical.dmi and put all the medical items items.dmi in there.
2. Old sprites were reused from the harvest items of comfrey and aloe (not the same item, new sprites do not belong in harvest.dmi at all).

Comment (Henri215) [reply code/game/objects/items/stacks/medical_packs.dm:274]: If you really want to move all the medical sprites here then i would suggest you to spawn all of them to test, i think you you forgot to change the icon path in some items.

**Review by Henri215 — COMMENTED** [review]
gauze and ointment shouldnt go to medical.dmi too?

**Comment by Fordoxia** [comment]
> gauze and ointment shouldnt go to medical.dmi too?

They are stacks, and all stackable objects live in their own group of folders without exception, having checked through every single DMI.

**Review by Henri215 — COMMENTED** [review]
image

Ointment was probably added here by accident since you didnt remove it from items.dmi

**Comment by Fordoxia** [comment]
> image
> 
> Ointment was probably added here by accident since you didnt remove it from items.dmi

I think I actually forgot to push my last commit hahaha

**Comment by Henri215** [comment]
tmp/dummy-save-1.sav? what are those?

**Comment by Henri215** [comment]
image

That ointment on items.dmi was a duplicate?

**Comment by Fordoxia** [comment]
> tmp/dummy-save-1.sav? what are those?

I have no idea what those are or how they got there. They have been purged.

**Comment by Fordoxia** [comment]
> image
> 
> That ointment on items.dmi was a duplicate?

As far as I can tell, yes, it was a duplicate the whole time.

**Comment by Henri215** [comment]
> > image
> > That ointment on items.dmi was a duplicate?
> 
> As far as I can tell, yes, it was a duplicate the whole time.

Then the one you added to medical.dmi can be deleted?

**Comment by Fordoxia** [comment]
> > > image
> > > That ointment on items.dmi was a duplicate?
> > 
> > 
> > As far as I can tell, yes, it was a duplicate the whole time.
> 
> Then the one you added to medical.dmi can be deleted?

Yep.

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by S34NW — APPROVED** [review]

**Pull request merged**